tests/ci/select_circle_tests.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Select which tests to run on circleci.
+ *
+ * If CIRCLE_NODE_TOTAL and CIRCLE_NODE_INDEX environment vars are defined,
```

Comment:
The difference between longest and shortest build time is about 3mins, which I think is pretty good!

<img width="1087" alt="screen shot 2016-10-19 at 10 09 09 am" src="https://cloud.githubusercontent.com/assets/1295288/19522182/43a31524-95e4-11e6-82a0-37321ba1cb0c.png">